docs/tutorial/README.md

```diff
@@ -0,0 +1,252 @@
+
+`./scripts/3-deploy-demo/2-deploy-demo.sh`
+
+### 3.3. Deploy Azure Id to Kubernetes
+
+We need to tell the cluster about the Id we created, so it can bind it to the pod (the next step). To do that, we will deploy the spec found in `/deploy/demo/aadpodidentity.yaml`. You need to set the following values in that file:
+
+- metadata->name: A unique name we will use in the next step to reference this Id.
+- type: 0=Azure Identity; 1=Service Principal. You should set it to 0.
+- ResourceID: subscriptionId, resource group, and id name. You can find this full string by running `az identity list` and looking at the `id` field.
+- ClientID: Client Id of the Azure ID. You can find this full string by running `az identity list` and looking at the `clientId` field.
+
+Once those values are set, you can deploy the identity with the following
+
+`./scripts/3-deploy-demo/3-deploy-id-to-k8s.sh`
+
+### 3.4. Bind the Id to our demo pod
+
+Last thing we need to do is bind the Id we created in step 1, and deployed in step 3, to the pod we deployed in step 2.
+
+Edit the file `/deploy/demo/aadpodidentitybinding.yaml` with the following values:
+
+- azureIdentity: the name for the AzureIdentity you deployed in the previous step, which can remain `demo-aad1`
+- selector: in our case, keep the value as `demo`.
+
+Deploy the binding with the following
+
+`./scripts/3-deploy-demo/4-deploy-id-binding.sh`
+
+## Did it work?
+
+You'll need to check the logs of each pod to know if everything worked.
+
+First, get the pod names with the following command:
+
+```sh
+
+$ kubectl get pods
+NAME                   READY     STATUS    RESTARTS   AGE
+demo-757967c54-64pzr   1/1       Running   0          1h     # the demo pod
+mic-64ddcf5f65-h4hft   1/1       Running   0          19h    # the MIC pod
+nmi-b9xbg              1/1       Running   0          1h     # the NMI pod
+
+```
+
+### Check the Managed Identity Controller pod
+
+Check the logs of the MIC controller and see the binding successfully applied on the node.
+
+```sh
+
+$ kubectl logs mic-64ddcf5f65-h4hft
+....
+I0606 23:19:45.867711       1 crd.go:123] Got id podid to assign
+I0606 23:19:45.867829       1 crd.go:142] Creating assigned Id: demo-5788d95785-ghzwv-default-podid
+I0606 23:19:45.874002       1 cloudprovider.go:170] Find aks-nodepool1-15831963-0 in resource group: MC_k8s-test_clusterFrank_eastus
+I0606 23:20:11.051552       1 cloudprovider.go:162] Underlying cloud provider operation took 25.04421296s
+I0606 23:20:11.051846       1 mic.go:259] Sync took: 25.220821436s
+I0606 23:20:11.052905       1 event.go:218] Event(v1.ObjectReference{Kind:"AzureIdentityBinding", Namespace:"default", Name:"myIdBinding", UID:"19a07e0e-69e0-11e8-9e9f-4addade2df92", APIVersion:"aadpodidentity.k8s.io/v1", ResourceVersion:"89529", FieldPath:""}): type: 'Normal' reason: 'binding applied' Binding myIdBinding applied on node aks-nodepool1-15831963-0 for pod demo-5788d95785-ghzwv-default-podid
+
+```
```

Comment:
We can see the same events here as well, either:

kubectl get events --field-selector=involvedObject.kind=AzureIdentityBinding

or 

kubectl describe azureidentitybinding

Reply:
`kubectl get events --field-selector=involvedObject.kind=AzureIdentityBinding` didn't work for me

but I added docs saying to try the describe command.